Resources/Prototypes/Floof/Entities/Objects/Specific/Robotics/endoskeleton.yml

```
@@ -0,0 +1,153 @@
+      borg_head+o:
+        whitelist:
+          tags:
+          - BorgGenericHead
+      borg_chest+o:
+        whitelist:
+          tags:
+          - BorgGenericTorso
+      borg_l_salvarm+o:
+        whitelist:
+          tags:
+            - BorgMiningLLeg
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 10 but found 12

(indentation)